src/ImageSearchModal.jsx

```diff
@@ -110,6 +116,15 @@ export class ImageSearchModal extends React.Component {
     render() {
         let defaultBody = (
             <React.Fragment>
+                <form className="ct-form">
+                    <label className="control-label" htmlFor="as-user">{_("Download as:")}</label>
+                    <fieldset id="as-user">
+                        <input type="radio" value="user" id="user" onChange={this.onToggleUser} checked={!this.state.isRoot} />
+                        <label className="radio" htmlFor="user">{_("user")}</label>
+                        <input type="radio" value="root" id="root" onChange={this.onToggleUser} checked={this.state.isRoot} />
+                        <label className="radio" htmlFor="root">{_("root")}</label>
```

Comment:
Same "root" vs. "System" thing here. That may need @garrett 's opinion. Also, it feels like these words  should be capitalized.

Reply:
I went for 'System' and real user name. So in my case `mmarusak` as it is the same as what is shown later on as `owner`.